package.json

```diff
@@ -103,5 +103,8 @@
     "@storybook/react": "^6.1.15",
     "@types/react-router-dom": "^5.1.6",
     "ts-jest": "^26.4.4"
+  },
+  "resolutions": {
+    "babel-loader": "8.1.0"
```

Comment:
I ran into this issue, seems to me the resolver isn't needed with the react storybook upgrade. Mind upgrading react storybook & trying to remove this & seeing if `yarn && yarn start ` works?

Reply:
No luck upgrading storybook, we get the same error.

Reply:
https://github.com/storybookjs/storybook/issues/5183

Here's a very long issue, but there's no real fix yet.  The workarounds are to add `resolutions` to package.json as we have done here, or just to silence the warning.